README.md

```diff
@@ -1,15 +1,126 @@
+    - Timestamps are appended with `_at`
+    - ID primary keys are prefixed with the name of the table.  For example, the `campaign` table's ID column is renamed `campaign_id`.
+    - Foreign keys include the table that they refer to. For example, a campaign's `list` ID column is renamed `list_id`.
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
+## Installation Instructions
+Add the following to your `packages.yml` file:
+```yml
+# packages.yml
+packages:
+  - package: fivetran/mailchimp_source
+    version: [">=0.3.0", "<0.4.0"]
+```
+
+Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+
+## Configuration
+By default, this package looks for your Mailchimp data in the `mailchimp` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Mailchimp data is, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+    mailchimp_database: your_database_name
+    mailchimp_schema: your_schema_name 
+```
+
+
+### Passthrough Columns
+
+This package includes all of the source columns that are defined in the macros folder. We recommend including custom columns in this package because the staging models only bring in the standard columns for the `MEMBER` table.
+
+You can add more columns using our passthrough column variables. These variables allow the passthrough columns to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure datatype casting by using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - we rename this column with the alias attribute - and your custom passthrough columns will be casted accordingly.
+
+Use the following format for declaring the respective passthrough variables (note to call the alias name, not original name, in the transform_sql):
+
+```yml
+# dbt_project.yml
+
+vars:
+
+  mailchimp__members_pass_through_columns:
+    - name:           "custom_field_name"
+      alias:          "normal_field_name"
+      transform_sql:  "cast(normal_field_name as string)"
+```
+
+
+### Disabling models
+
+It's possible that your Mailchimp connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Mailchimp or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be `true`. Add variables for only the tables you would like to disable:  
```

Comment:
There are a number of models that are able to be disabled from this variable. Would you be able to list out all the source tables that would be ignored if the users ads this variable config to `true`.

Reply:
Can you confirm this is what you have in mind when listing out tables?
```


### Disabling models

It's possible that your Mailchimp connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Mailchimp or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be `true`. Add variables for only the tables you would like to disable. The following source tables will be disabled if the vars is set to true:

- automation
- automation_email
- automation_recipient
- automation_recipient_activity

```yml
# dbt_project.yml

...
config-version: 2

vars:
    using_automations: false   # Disable if you are not using automations 
    
```

Reply:
Also, re: list out all the source tables that would be ignored if the users ads this variable config to true -- shouldn't it be false?